peas/tests/test_sensors.py

```diff
@@ -5,7 +5,7 @@
 import serial
 
 from peas import sensors as sensors_module
-#from peas.sensors import ArduinoSerialMonitor
+from peas.sensors import ArduinoSerialMonitor
```

Comment:
I don't see this used. Is it coming in your upcoming PR?

Reply:
Actually, I show that a few of these modules aren't used. One of my plugins highlights them:

![unused-modules](https://user-images.githubusercontent.com/70736/35028961-b755e8dc-fbac-11e7-825c-13640a930ea3.png)

Reply:
Please let me know which plugin that is.

And I'll remove the currently unnecessary imports.